frontend/plugins/resources/src/components/ResourceTabs/ResourceTabs.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { HeaderTabs } from '@backstage/core';
+import { useNavigate } from 'react-router-dom';
+import { ResourcesTabsLabels } from '../App'
+
+const ResourceTabs = () => {
+  const navigate = useNavigate()
+
+  const doChange = (idx: number): void => {
+    const tab = ResourcesTabsLabels.find(x => x.id === idx)
+    if(tab) {
```

Comment:
_nitpick:_ I’d add a space before the bracket

Reply:
fixed